lib/gui/etcher.js

```diff
@@ -83,6 +85,25 @@ const createMainWindow = () => {
   } else {
     mainWindow.loadURL(`file://${path.join(__dirname, 'app', 'index.html')}`)
   }
+
+  const page = mainWindow.webContents
+  console.log('setting updater')
+
+  page.once('did-frame-finish-load', () => {
+    autoUpdater.on('checking-for-update', () => {
+      console.log('checking-for-update')
+    })
+    autoUpdater.on('update-available', (info) => {
+      console.log('update-available', info)
+    })
+    autoUpdater.on('update-not-available', (info) => {
+      console.log('update-not-available', info)
+    })
+    autoUpdater.on('error', (err) => {
+      console.log('error', err)
+    })
+    autoUpdater.checkForUpdatesAndNotify()
```

Comment:
Should this also respect the 'updatesEnabled' setting?

Reply:
Absolutely, which is why it's only a bunch of `console.log`s for now